php/libraries/NDB_Caller.class.inc

```diff
@@ -485,6 +505,50 @@ class NDB_Caller
         return $html;
     }
 
+    /**
+     * Renders html for a direct data entry Instrument
+     *
+     * @param string $instrumentName      Name of the instrument to load
+     * @param string $page                Page (subtest) of the instrument
+     * @param string $commentID           CommentID of the instrument to load
+     * @param string $redirectToOnSuccess URL to redirect to if the page is loaded
+     *                                    successfully.
+     *
+     * @return string HTML of the page to render.
+     */
+    function loadInstrumentDirect(
+        $instrumentName,
+        $page,
+        $commentID,
+        $redirectToOnSuccess=null
+    ) {
```

Comment:
Can you add `: String` to indicate the return type?

Reply:
🤷‍♂️ OK but it might help us catch errors.

Reply:
we can do it in a followup PR to minor